docs/reference/migration/migrate_5_0/search.asciidoc

```diff
@@ -128,6 +128,7 @@ in favour of `query` and `no_match_query`.
 
 * Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoPolygonQuery. Use parameter validation_method instead.
 
+* Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoDistanceRangeQuery. Use parameter validation_method instead.
```

Comment:
Was there no documentation for this querybuilder where we might add deprecation warnings? It would be good to explain the parameters also in the regular doc.